resources/fetching/condition/is_leader.go

```diff
@@ -15,30 +15,30 @@
 // specific language governing permissions and limitations
 // under the License.
 
-package conditions
+package condition
 
 import (
 	"github.com/elastic/cloudbeat/resources/fetching"
 	"github.com/elastic/cloudbeat/uniqueness"
 	"github.com/elastic/elastic-agent-libs/logp"
 )
 
-type LeaseFetcherCondition struct {
+type IsLeader struct {
```

Comment:
Is it according to Go conventions? the previous name is better in my opinion.

Reply:
I believe that the name `condition.IsLeader` is easier and more understandable than `condition.LeaseFetcherCondition`. Happy to rename though if you prefer it.

Reply:
Thanks, Let's keep it.